routes/middleware/post_place.js

```diff
@@ -1,38 +1,30 @@
-const request = require('request');
+const axios = require('axios');
 require('dotenv').config();
 
-module.exports = (req, res, next) => {
-  var url = `${process.env.URI}/places`;
+module.exports = async ({ body }, res, next) => {
+  const url = `${process.env.URI}/places`;
```

Comment:
You could even just pop this directly on the post. Not fussed either way though, I guess this is clearer.